docs/reference/evercrypt/dh/index.md

```diff
@@ -0,0 +1,38 @@
+# Diffie-Hellman
```

Comment:
We should use Elliptic-curve Diffie-Hellman everywhere. This could be confused with finite field DH.